src/functions/module_lookup.ts

```diff
@@ -528,10 +531,18 @@ export const XLOOKUP = {
       () => [-1, 1, -2, 2].includes(_searchMode),
       _t("searchMode should be a value in [-1, 1, -2, 2].")
     );
-    assert(() => [-1, 0, 1].includes(_matchMode), _t("matchMode should be a value in [-1, 0, 1]."));
+    assert(
+      () => [-1, 0, 1, 2].includes(_matchMode),
+      _t("matchMode should be a value in [-1, 0, 1, 2].")
```

Comment:
the argument is named `match_mode` for the user
```suggestion
      _t("match_mode should be a value in [-1, 0, 1, 2].")
```